src/cosmicqc/analyze.py

```diff
@@ -52,6 +54,10 @@ def identify_outliers(
             or not for use within other functions.
     """
 
+    # interpret the df as SCDataFrame
```

Comment:
Can you add a bit more detail to this comment? Why is it important that `df` is an `SCDataFrame` object?

Reply:
Thanks, for sure - roughly, I hope we can use methods from the class to help with CLI efforts through this function.